copernicusmarine/core_functions/documentation_utils 2.py

```diff
@@ -0,0 +1,256 @@
+        " the documentation or use option ``--create-template`` "
+        "for an example template."
+    ),
+    "MOTU_API_REQUEST_HELP": (
+        "Option to pass a complete MOTU API request as a string. Caution, user has to "
+        "replace double quotes “ with single quotes ‘ in the request."
+    ),
+    "NETCDF_COMPRESSION_LEVEL_HELP": (
+        "Specify a compression level to apply on the NetCDF output file. "
+        "A value of 0 means no compression, and 9 is the highest level of "
+        "compression available."
+    ),
+    "NETCDF_COMPATIBLE_HELP": (
+        "Enable downloading the dataset in a netCDF3 compatible format."
+    ),
+}
+
+GET = {
+    "GET_DESCRIPTION_HELP": (
+        "Download originally produced data files.\n \n  "
+        " The ``--dataset-id`` is required and can be found via the ``describe``"
+        " command."
+    ),  # has some hardcoding in CLI
+    "GET_RESPONSE_HELP": (
+        "ResponseGet \n"
+        "A list of files that were downloaded and some metadata."
+    ),
+    "REQUEST_FILE_HELP": (
+        "Option to pass a file containing the arguments. The file MUST follow the "
+        "structure of dataclass ‘GetRequest’. For more information please refer to"
+        " the documentation or use option ``--create-template``"
+        " for an example template."
+    ),
+    "SHOW_OUTPUTNAMES_HELP": (
+        "Option to display the names of the output files before download."
+    ),
+    "FILTER_WITH_GLOBBING_PATTERN_HELP": (
+        "A pattern that must match the absolute paths of the files to download."
+    ),
+    "FILTER_WITH_REGULAR_EXPRESSION_HELP": (
+        "The regular expression that must match the absolute paths of the files to "
+        "download."
+    ),
+    "FILE_LIST_HELP": (
+        "Path to a .txt file containing a list of file paths, line by line, that will "
+        "be downloaded directly. These files must be from the specified dataset using "
+        "the –dataset-id. If no files can be found, the Toolbox will list all files "
+        "on the remote server and attempt to find a match."
+    ),
+    "CREATE_FILE_LIST_HELP": (
+        "Option to only create a file containing the names of the targeted files "
+        "instead of downloading them. It writes the file in the directory specified "
+        "with the –output-directory option (default to current directory). The file "
+        "name specified should end with ‘.txt’ or ‘.csv’. If specified, no other "
+        "action will be performed."
+    ),
+    "SYNC_HELP": (
+        "Option to synchronize the local directory with the remote directory. See the "
+        "documentation for more details."
+    ),
+    "SYNC_DELETE_HELP": (
+        "Option to delete local files that are not present on the remote server while "
+        "applying sync."
+    ),
+    "INDEX_PARTS_HELP": (
+        "Option to get the index files of an INSITU dataset. Temporary option."
```

Comment:
```suggestion
    "INDEX_PARTS_HELP": (
        "Option to get the index files of an INSITU dataset."
```
Let's just deprecate it if we want to delete it at some point

Reply:
Mmh then I should add it in the list of deprecated options too, right?

Reply:
Sorry: Let's just deprecate when we start planning to delete it